mobile-app/app/screens/AppNavigator/screens/Auctions/components/BrowseAuctions.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import { tailwind } from '@tailwind'
+import { ThemedScrollView } from '@components/themed'
+import { BatchCard } from '@screens/AppNavigator/screens/Auctions/components/BatchCard'
+import { useFeatureFlagContext } from '@contexts/FeatureFlagContext'
+import { View } from 'react-native'
+import { InfoText } from '@components/InfoText'
+import { translate } from '@translations'
+import { useDispatch, useSelector } from 'react-redux'
+import { RootState } from '@store'
+import { useEffect } from 'react'
+import { fetchVaults } from '@store/loans'
+import { useWhaleApiClient } from '@shared-contexts/WhaleContext'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { SkeletonLoader, SkeletonLoaderScreen } from '@components/SkeletonLoader'
+import { LoanVaultLiquidated, LoanVaultLiquidationBatch } from '@defichain/whale-api-client/dist/api/loan'
+
+export function BrowseAuctions (): JSX.Element {
+  const dispatch = useDispatch()
+  const client = useWhaleApiClient()
+  const { address } = useWalletContext()
+  const blockCount = useSelector((state: RootState) => state.block.count)
+  const auctions = useSelector((state: RootState) => state.auctions.auctions)
+
+  useEffect(() => {
+    dispatch(fetchVaults({ address, client }))
+  }, [blockCount])
+
+  const { isBetaFeature } = useFeatureFlagContext()
+
+  return (
+    <ThemedScrollView style={tailwind('h-full m-4')} testID='auctions_cards'>
```

Comment:
```suggestion
    <ThemedScrollView contentContainerStyle={tailwind('p-4')} testID='auctions_cards'>
```
Recommend to use `contentContainerStyle` if the intention is to apply styling on the container instead of the child items.

By convention, if you apply **margin** on a parent container, one would expect the spacing is applied **outside** of the scroll view, but actually you want to apply **padding** inside of the **container**

Reply:
Will follow this Thanks